generative_ai/gemini_safety_config_example.py

```diff
@@ -23,23 +23,17 @@ def generate_text() -> str:
 
     from vertexai.generative_models import (
         GenerativeModel,
-        GenerationConfig,
         HarmCategory,
         HarmBlockThreshold,
         Part,
-        SafetySetting
+        SafetySetting,
     )
 
     # TODO(developer): Update project
     vertexai.init(project=PROJECT_ID, location="us-central1")
 
     model = GenerativeModel("gemini-1.5-flash-001")
 
-    # Generation config
```

Comment:
(request) Let us drop this! Using Safety config with temperate, top_p and top_k make a lot of sense me during content generation. 

Also, other code samples are are also having same format.